lib/rucio/daemons/replicarecoverer/suspicious_replica_recoverer.py

```diff
@@ -524,7 +577,15 @@ def run_once(heartbeat_handler: Any, younger_than: int, nattempts: int, vos: Opt
     return must_sleep
 
 
-def run(once: bool = False, younger_than: int = 5, nattempts: int = 5, vos: list[str] = None, limit_suspicious_files_on_rse: int = 5, json_file_name: str = "/opt/rucio/etc/suspicious_replica_recoverer.json", sleep_time: int = 3600, active_mode: bool = False) -> None:
+def run(
+        once: bool = False,
+        younger_than: int = 5,
+        nattempts: int = 5,
+        vos: Optional[list[str]] = None,
```

Comment:
```suggestion
        vos: Optional[Sequence[str]] = None,
```

Imported from `collections.abc`